modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -39,15 +39,15 @@ spec:
           pod: {}
         - name: bridge-net <1>
           multus:
-            networkName: <a-bridge-network> <2>
+            networkName: <network-namespace/a-bridge-network> <2>
 ...
 ----
 <1> The name of the bridge interface. This value must match the `name` value of the corresponding `spec.template.spec.networks` entry.
 <2> The name of the network attachment definition.
 +
 [NOTE]
 ====
-The example virtual machine is connected to both the `default` pod network and `bridge-net`, which is defined by a network attachment definition named `a-bridge-network`.
+The example virtual machine is connected to both the `default` pod network and `bridge-net`, which is defined by a network attachment definition named `a-bridge-network`. However, if the network attachment definition is not in the same namespace as the VM, you can use a combination of the namespace and the network as the name of the network attachment definition.
```

Comment:
It took me a little while to figure out what you meant by this sentence. You might want to consider wording it like this:

"However, if the network attachment definition is not in the same namespace as the VM, you can combine the network namespace and the network name to create the network attachment definition name."

Or should it be "you must combine..."? Will there be a problem if the user does not combine the namespace and network name?

Reply:
It's not a "must" type thing. From what the original bug says, Dev just wants to gives users the option to combine the namespace and network name if the network attachment definition isn't the same namespace.

But the rewording is a good suggestion to make it cleaner.

Reply:
I know that this has been through QE but the bug sounds like this naming convention is required:

Description of problem:
We can not reference the NAD definations at a different   namespace of a VM 
...
Steps to Reproduce:
1. Create a VM at a namaspace
2. Try to reference a NAD defination at a different namespace. No parameters available to do so

It sounds like that is the only way the user can create a NAD for a network in a different namespace. I recommend asking the SME or QE whether this is optional. My gut feeling is that the NAD will not work if the network is in a different namespace and the network namespace is not part of the NAD.

Reply:
I don't follow. @apinnick is the point that the current text is wrong, that the feature should not be there or that it is simply vague (and we need to define the naming convention).

Reply:
@phoracek  My point is this: Is this naming convention optional or mandatory for referencing a network in a different namespace from the VM?

The wording "you can use a combination ..." suggests that it is optional. If it is mandatory, it might be better to phrase this as "you must use a combination ..."

Reply:
I see now. There is no other way to reference a network in another namespace. So mandatory.

Reply:
@mgarrellRH  On the basis of Petr's comment, I suggest this rewording to reflect that this naming convention is mandatory. 

Also, the first use of "network attachment definition" in your original version does not seem to be correct. Since the NAD defines the network attachment, it is probably the network that is in a different namespace and not the NAD itself. See Petr's last comment, "There is no other way to reference a network in another namespace."

Suggested rewrite:
"If the network is not in the same namespace as the VM, you must include the network namespace in the network attachment definition." or "you must combine the network namespace and network name to create..."

Reply:
@phoracek @apinnick "However, if the network attachment definition is not in the same namespace as the VM, you must combine the namespace and the network name to create the network attachment definition name."

Does this rewrite meet with everyone's approval?

Reply:
I still think the first use of network attachment definition is incorrect but I will not hold up the doc merge.

Reply:
Sounds fine to me